src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -6162,70 +6162,29 @@ export class PostgresCohortMembersService {
-   * Makes API call to LMS service to create enrollment
-   *
-   * @param userId - The user ID to enroll
-   * @param courseId - The course ID to enroll in
-   * @returns Promise with enrollment result
-   */
-  private async enrollUserToSingleCourse(
-    userId: string,
-    courseId: string,
-    cohortId: string
-  ) {
+  ): Promise<Array<{ courseId: string; status: 'success' | 'failed'; result?: any; error?: any }>> {
```

ESLint

[error] Replace `Array<{·courseId:·string;·status:·'success'·|·'failed';·result?:·any;·error?:·any·}>` with `⏎····Array<{⏎······courseId:·string;⏎······status:·"success"·|·"failed";⏎······result?:·any;⏎······error?:·any;⏎····}>⏎··`

(prettier/prettier)